web_dialog_size/static/src/js/web_dialog_size.js

```diff
@@ -25,15 +25,16 @@ Dialog.include({
     open: function() {
         var res = this._super.apply(this, arguments);
         this.$modal.draggable({
-            handle: "div.modal-header",
+            handle: '.modal-header',
+            helper: false
         });
         return res;
     },
 
     close: function() {
-        var draggable = this.$modal.draggable( "instance" );
+        var draggable = this.$modal.draggable('instance');
         if (draggable)
-            this.$modal.draggable("destroy");
+            this.$modal.draggable('destroy');
```

Comment:
Leave this as it was please.

Reply:
I think this is because of the JS linter

Reply:
Then the linter should be fixed.

Reply:
OCA has no preference in that, so the best is not to change lines unless required. 😊

Reply:
@Yajo Ok, but anyway isn't it better to not have them mixed?

Reply:
Of course, I only mean to leave them as they are if you don't need to change them. 😉